src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -207,26 +207,913 @@
+                            <Grid>
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="1*" />
+                                    <ColumnDefinition Width="1*" />
+                                </Grid.ColumnDefinitions>
+                                <Button Grid.Column="0" x:Uid="GeneralPage_BackupConfigs"
+                                        Command="{Binding BackupConfigsEventHandler}"
+                                        IsEnabled="True"
+                                        Visibility="Visible"
+                                        HorizontalAlignment="Right"/>
+
+                                <Button Grid.Column="2" x:Uid="GeneralPage_RestoreConfigs"
+                                        Command="{Binding RestoreConfigsEventHandler}"
+                                        IsEnabled="True"
+                                        Visibility="Visible"
+                                        HorizontalAlignment="Right"/>
+
+                            </Grid>
+
+                        </controls:Setting.ActionContent>
+
+
+
+                    </controls:Setting>
+
+
+
+                    <!-- SettingsBackupWasSuccessful -->
+                    <InfoBar x:Uid="General_SettingsBackupMessageResults"
+                                  IsClosable="False"
+                                  Severity="Success"
+                                  IsTabStop="{Binding SettingsBackupWasSuccessful, Mode=OneWay}"
+                                  IsOpen="{Binding SettingsBackupWasSuccessful, Mode=OneWay}"
+                                  Message="{Binding SettingsBackupMessage}"/>
+
+                    <!-- SettingsBackupWasUnsuccessful  -->
+                    <InfoBar x:Uid="General_SettingsBackupMessageResults"
+                                  IsClosable="False"
+                                  Severity="Warning"
+                                  IsTabStop="{Binding SettingsBackupWasUnsuccessful, Mode=OneWay}"
+                                  IsOpen="{Binding SettingsBackupWasUnsuccessful, Mode=OneWay}"
+                                  Message="{Binding SettingsBackupMessage}">
+                    </InfoBar>
+
+                    <!-- GeneralPage_ToggleSwitch_AutoAutoBackup  -->
+                    <controls:Setting x:Uid="GeneralPage_ToggleSwitch_AutoDownloadUpdates"
+                                      Margin="0,-6,0,0"
+                                      IsEnabled="True"                                       
+                                      Visibility="Collapsed">
+                        <controls:Setting.ActionContent>
+                            <ToggleSwitch IsOn="{Binding Mode=TwoWay, Path=AutoDownloadUpdates}" x:Uid="ToggleSwitch"/>
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+
+                    <!--<controls:Setting x:Uid="ColorModeHeader" Icon="&#xE790;">
```

Comment:
why is this commented?

Reply:
I really don't know, it's not now though. It should not have been commented out.